includes/Revision.php

```diff
@@ -1,4 +1,5 @@
 <?php
+use Wikia\Util\PerformanceProfilers\UsernameUseProfiler;
 
 /**
  * @todo document
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1135)